code/game/objects/effects/anomalies.dm

```diff
@@ -53,20 +60,18 @@
 
 /obj/effect/anomaly/grav/anomalyEffect()
 	..()
-
 	boing = 1
 	for(var/obj/O in orange(4, src))
 		if(!O.anchored)
 			step_towards(O,src)
 	for(var/mob/living/M in orange(4, src))
 		step_towards(M,src)
-	for(var/obj/O in orange(1,src))
+	for(var/obj/O in range(0,src))

```

Comment:
```orange()``` was a lazy way to make it not throw itself around.
if you don't want this, make sure to check for ```O``` being ```src```